samtools.wdl

```diff
@@ -56,18 +52,23 @@ task Index {
 task Merge {
     input {
         Array[File]+ bamFiles
-        String outputBamPath
+        String outputBamPath = "merged.bam"
```

Comment:
Not sure this should have a default, either.

Reply:
Same comment as above. This allows you to use the task and take the outputBam. No need to worry where it is stored.